Assets/Scripts/TES/MorrowindEngine.cs

```diff
@@ -551,9 +556,10 @@ private GameObject InstantiateLight(LIGHRecord LIGH, bool indoors)
 			lightComponent.range = 3 * (LIGH.LHDT.radius / Convert.meterInMWUnits);
 			lightComponent.color = new Color32(LIGH.LHDT.red, LIGH.LHDT.green, LIGH.LHDT.blue, 255);
 			lightComponent.intensity = 1.5f;
+			lightComponent.bounceIntensity = 0f;
 			//lightComponent.shadows = LightShadows.Soft;
 
-			if(!indoors)
+			if(!indoors & false)//disabling exterior cell lights because there is no day/night cycle
```

Comment:
That's correct. I was using this temporarily but I've already added the option to the local settings so you can can toggle it from there instead.